src/lib/stitching/gravity/stitching.ts

```diff
@@ -104,6 +104,7 @@ export const gravityStitchingEnvironment = (
       extend type ArtistSeries {
         artists(page: Int, size: Int): [Artist]
         image: Image
+        artworksConnection(first: Int, after: String): ArtworkConnection
```

Comment:
Looks good — but I wonder can we add field descriptions to stitched fields? Might help avoid some dev confusion in the future between these two artworks-related connection fields